.teamcity/_self/projects/WebApp.kt

```diff
@@ -477,30 +512,7 @@ fun seleniumBuildType( viewportName: String, buildUuid: String): BuildType  {
 					# mocha-teamcity-reporter to work.
 					export MAGELLANDEBUG=true
 
-					IMAGE_URL="https://calypso.live?image=registry.a8c.com/calypso/app:build-${BuildDockerImage.depParamRefs.buildNumber}";
-					MAX_LOOP=10
-					COUNTER=0
-
-					# Transform an URL like https://calypso.live?image=... into https://<container>.calypso.live
-					while [[ ${'$'}COUNTER -le ${'$'}MAX_LOOP ]]; do
-						COUNTER=${'$'}((COUNTER+1))
-						REDIRECT=${'$'}(curl --output /dev/null --silent --show-error  --write-out "%{http_code} %{redirect_url}" "${'$'}{IMAGE_URL}")
-						read HTTP_STATUS URL <<< "${'$'}{REDIRECT}"
-
-						# 202 means the image is being downloaded, retry in a few seconds
-						if [[ "${'$'}{HTTP_STATUS}" -eq "202" ]]; then
-							sleep 5
-							continue
-						fi
-
-						break
-					done
-
-					if [[ -z "${'$'}URL" ]]; then
-						echo "Can't redirect to ${'$'}{IMAGE_URL}" >&2
-						echo "Curl response: ${'$'}{REDIRECT}" >&2
-						exit 1
-					fi
+					URL="%dep.calypso_BuildDockerImage.calypso_live_url%"
```

Comment:
Nice!

Reply:
Edit: Unfortunately this change is breaking some things:

When we build the docker image, we compute the calypso.live URL and transform it from `https://calypso.live?image=<image>` to `https://<container>.calypso.live`. Every time we run E2E we will use the `<container>` form. However, this container is disposed after 1h of inactivity. Rerunning an E2E build whose Docker image was created more than 1h ago will result in it using a `<container>` URL that doesn't exists anymore.

Reply:
Interesting, I didn't realize that!